fastify.js

```diff
@@ -417,15 +417,19 @@ function build (options) {
   }
 
   function _route (_fastify, method, url, options, handler) {
-    if (!handler && typeof options === 'function') {
-      handler = options
-      options = {}
+    if (!handler) {
+      if (typeof options === 'function') {
+        handler = options
+        options = {}
+      } else if (typeof options !== 'object') {
```

Comment:
`typeof []` => `'object'`

`} else if (Object.prototype.toString.call(options) !== '[object Object]') {`

Reply:
Thanks, you are right, this code is more rigorous.

Reply:
I already update the tests.